### PR TITLE
chore(db): add dummy items

### PR DESCRIPTION
**Pull request by tgrigoruk**
### Summary
List of changes introduced by this pull request

### Retrospective
Reflection of things learned or hardships encountered -- used as learning resource

### Testing Notes
Notes for the reviewer of things to be aware of during testing

### Testing
Comprehensive list of instructions to verify the changes are working as intended

### CODE REVIEW

File: apps/api/src/models/dummy.ts [hunk 1]
```diff
@@ -248,25 +248,28 @@ const DUMMY_USERS: UsersType[] = [
 
 const tagsText: string[] = [
```
Comment (rhysmah) [inline apps/api/src/models/dummy.ts:249]: These tags make sense to me.

File: apps/api/src/models/dummy.ts [hunk 2]
```diff
@@ -333,11 +336,11 @@ const DUMMY_ITEMS: ItemsType[] = [
...
-    brand: 'Warner Bros.',
-    price: 9999, // in cents ($99.99)
+    title: 'Batman: The Dark Knight Returns Graphic Novel',
+    description: 'Classic graphic novel by Frank Miller featuring an older Bruce Wayne returning to the role of Batman.',
+    imageUrl: 'https://example.com/batman-dark-knight-returns.jpg',
+    brand: 'DC Comics',
+    price: 1999, // in cents ($19.99)
     isForSale: true,
     isForTrade: true,
   },
```
Comment (rhysmah) [inline apps/api/src/models/dummy.ts:346]: This makes sense. The ItemCard I created is pretty simple: it has only a title, image, and description.

In the wireframes, the modal shows only a little more information, namely tags. So I'll display those.

Perhaps we can add an optional field, 'itemURL', that applies wishlists but not to collections.

**Review by rhysmah — APPROVED** [review]
The dummy data looks great. For the frontend, unless we decide differently, I only really need a title, description, image, tags, and possibly a URL (for wishlist items).

File: apps/api/src/models/dummy.ts (hunk 2 above)
Comment (tgrigoruk) [reply apps/api/src/models/dummy.ts:346]: I actually haven't finished defining `DUMMY_ITEMS_TAGS` (underneath `DUMMY_ITEMS`). That would populate the join table that stores items' tags. These tags will be returned on an item query i.e. GET `/item/:itemId`, so expect a field like `tags: ["Star Wars", "Marvel"]` in the response.

`itemUrl` is probably necessary, i'll add that.

btw i'll be changing price to be 2-places decimal.

**Pull request merged**